app/models/validations/setup_validations.rb

```diff
@@ -72,19 +72,15 @@ def validate_irproduct_other(record)
     end
   end
 
-  def validate_scheme_has_confirmed_locations_validation(record)
-    return unless record.scheme
-
-    unless record.scheme.locations.confirmed.any?
-      record.errors.add :scheme_id, :no_completed_locations, message: I18n.t("validations.lettings.setup.scheme.no_completed_locations")
-    end
-  end
-
   def validate_scheme(record)
     if record.scheme&.status == :incomplete
       record.errors.add :scheme_id, :incomplete, message: I18n.t("validations.lettings.setup.scheme.incomplete")
     end
 
+    if record.scheme&.locations.present? && record.scheme.locations.confirmed.none?
```

Comment:
This makes sense for enforcing order, but does it not also want to show if `record.scheme.locations.present?` is false?